testonly/hammer/hammer.go

```diff
@@ -252,6 +253,35 @@ func HitMap(ctx context.Context, cfg MapConfig) error {
 	return firstErr
 }
 
+// mapWorker allows each thread to have its own PRNG, which makes it easy to
```

Comment:
nit: "thread" sounds a bit too generic, and is also not a Golang term, so triggers "what's that"s. Do you have a mental model of what `mapWorker` will do in follow-ups? The "worker" term might be good enough already.

Reply:
Done.